extensions/odin/src/views/odin-alarms-view.tsx

```diff
@@ -0,0 +1,63 @@
+import { List } from "@raycast/api";
+import { useEffect, useState } from "react";
+import OdinHelper from "../helpers/OdinHelper";
+import { OdinAlarm } from "../models/OdinAlarm";
+import { OdinAlarmListItem } from "./odin-alarm-list-item";
+import {
+  ODIN_SOURCE_INDICATION,
+  ODIN_STRINGS_NO_ALARMS_FOUND,
+  ODIN_STRINGS_SEARCH_PLACEHOLDER,
+} from "../constants/OdinConstants";
+
+export default function OdinAlarmsView() {
+  const odinHelper = new OdinHelper();
+
+  const [state, setState] = useState<{
+    odinAlarmsModel: null | [OdinAlarm];
+    lastUpdated: null | string;
+    isLoading: boolean;
+  }>({
+    odinAlarmsModel: null,
+    lastUpdated: null,
+    isLoading: true,
+  });
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const retrievedResult = await odinHelper.retrieveAlarmsFromOdinPuls();
+
+        setState((prevState) => {
+          return {
+            ...prevState,
+            odinAlarmsModel: retrievedResult?.[0] as [OdinAlarm],
+            lastUpdated: retrievedResult?.[1] as string,
+            isLoading: false,
+          };
+        });
+      } catch (error: unknown) {
+        console.log(error);
+      }
+    }
+
+    fetchData();
+  }, []);
+
+  return (
+    <>
+      <List
+        isLoading={state.isLoading}
+        searchBarPlaceholder={ODIN_STRINGS_SEARCH_PLACEHOLDER}
+        navigationTitle={`${ODIN_SOURCE_INDICATION} ${state.lastUpdated}`}
```

Comment:
```suggestion
```
Don't change the navigationTitle in the root command - it will be automatically set to the command name. Use navigationTitle only in nested screens to provide additional context. 

What you can do here is to set it as a [List.Section](https://developers.raycast.com/api-reference/user-interface/list#list.section) instead

Reply:
Done, check https://github.com/raycast/extensions/pull/2645/commits/30f6c8237861478052b225b440ff7970920d2114.